### PR TITLE
feat(mirrorbits-lite) use a volume for logs

### PR DESCRIPTION
**Pull request by dduportal**
Cherry pick of #730 and #732 to `mirrorbits-lite`

Includes an update of the chart metadatas

### CODE REVIEW

File: charts/mirrorbits-lite/Chart.yaml [hunk 1]
```diff
@@ -1,4 +1,7 @@
-apiVersion: v1
```
Comment (lemeurherve) [inline charts/mirrorbits-lite/Chart.yaml:1]: apiVersion is mandatory AFAIK

**Review by lemeurherve — APPROVED** [review]

File: charts/mirrorbits-lite/Chart.yaml [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+appVersion: v0.5.2
```
Comment (lemeurherve) [inline charts/mirrorbits-lite/Chart.yaml:1]:
```suggestion
apiVersion: v1
appVersion: v0.5.2
```

Comment (lemeurherve) [reply charts/mirrorbits-lite/Chart.yaml:1]:
```suggestion
```
And appVersion isn't (nit)

Comment (dduportal) [reply charts/mirrorbits-lite/Chart.yaml:1]: Good catch for the confusion I had between `appVersion` and `apiVersion` Thanks!

`appVersion` indicate the version of the application provided, which is (and should) always be different than the version of the Docker image. In our case it is definitively different as the version schem of jenkinsciinfra/mirrorbits is different than mirrorbits version scheme

Comment (dduportal) [reply charts/mirrorbits-lite/Chart.yaml:2]: Fixed by https://github.com/jenkins-infra/helm-charts/pull/731/commits/d9f4998bc4b6485670cf494d39d4c4cc4b5c793b

File: charts/mirrorbits-lite/Chart.yaml [hunk 3]
```diff
@@ -1,4 +1,8 @@
 apiVersion: v1
 description: Mirrobits lite helm chart for Kubernetes
 name: mirrorbits-lite
-version: 0.0.2
+version: 0.1.0
+appVersion: v0.5.2
```
Comment (lemeurherve) [inline charts/mirrorbits-lite/Chart.yaml:5]:
```suggestion
```
I still think there is zero added value with this (nit)

**Review by lemeurherve — APPROVED** [review]
LGTM

**Review by lemeurherve — APPROVED** [review]

File: charts/mirrorbits-lite/Chart.yaml (hunk 3 above)
Comment (dduportal) [reply charts/mirrorbits-lite/Chart.yaml]: It allows readers to immediately understand that the image tag version scheme is different than the application.
It's recommended by Helm doc but optional and it does not change the behavior (and we have it on the former `mirrorbits` chart?

**Pull request merged**